samples/nrf9160/tcp_client/src/main.c

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <modem/lte_lc.h>
+#include <net/socket.h>
+
+#define TCP_IP_HEADER_SIZE 28
+
+static int client_fd;
+static struct sockaddr_storage host_addr;
+static struct k_delayed_work server_transmission_work;
+
+K_SEM_DEFINE(lte_connected, 0, 1);
+
+static void server_transmission_work_fn(struct k_work *work)
+{
+	int err;
+	char buffer[CONFIG_TCP_DATA_UPLOAD_SIZE_BYTES] = {"\0"};
+
+	printk("Transmitting TCP/IP payload of %d bytes to the ",
+	       CONFIG_TCP_DATA_UPLOAD_SIZE_BYTES + TCP_IP_HEADER_SIZE);
+	printk("IP address %s, port number %d\n",
+	       CONFIG_TCP_SERVER_ADDRESS_STATIC,
+	       CONFIG_TCP_SERVER_PORT);
+
+	err = send(client_fd, buffer, sizeof(buffer), 0);
+	if (err < 0) {
+		printk("Failed to transmit TCP packet, %d\n", errno);
```

Comment:
`close()` the socket here? I know it's a simple app, but given it's a sample I would prefer to avoid leaving dangling sockets.